doc/api/stream.md

```diff
@@ -3500,7 +3500,7 @@ changes:
 * `options` {Object}
   * `highWaterMark` {number} Buffer level when
     [`stream.write()`][stream-write] starts returning `false`. **Default:**
-    `16384` (16 KiB), or `16` for `objectMode` streams.
+    `65536` (64 KiB), or `16` for `objectMode` streams.
```

Comment:
Can we add a changes documentation entry mentioning this pr and the change of default value?